app/e2e/tests/landing/landing-side-nav.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { test, expect } from '@playwright/test';
+import { PlaywrightLandingPage } from '@local/common/pages/playwright-landing-page';
+
+test.describe('landing page side nav', () => {
+    test.use({ storageState: undefined });
+
+    test('displays valid unauthenicated routes', async ({ page }) => {
+        const landing = new PlaywrightLandingPage(page);
+        await landing.goto();
+        await landing.openSideNav();
+        await landing.see(await landing.locateSideNavText('About Us'));
+        await landing.see(await landing.locateSideNavText('Getting Started Guide'));
+        await landing.see(await landing.locateSideNavText('Organizer Guide'));
+        await landing.see(await landing.locateSideNavText('Moderator Guide'));
+        await landing.see(await landing.locateSideNavText('Participant Guide'));
```

Comment:
maybe landing.see(landing.UnauthenticatedSidebarContent)?

Reply:
I think the way I have it is a bit more clear to understand what is being checked and adding to the class is unnecessary, but if you really think I should change it I can.f

Reply:
My opinion on this isn't strong, feel free to leave as-is.